lib/options.nix

```diff
@@ -246,7 +246,7 @@ rec {
   showDefs = defs: concatMapStrings (def:
     let
       # Pretty print the value for display, if successful
-      prettyEval = builtins.tryEval (lib.generators.toPretty {} def.value);
+      prettyEval = builtins.tryEval (lib.generators.toPretty { depthLimit = 10; } def.value);
```

Comment:
IMHO: Let's disable throwing here explicitly, regardless of whether we want to throw by default or not.